composer.json

```diff
@@ -6,20 +6,17 @@
     "license": "BSD-3-Clause",
     "require": {
         "silverstripe/recipe-plugin": "^1",
-        "silverstripe/recipe-core": "4.5.x-dev",
-        "cwp/cwp-core": "2.5.x-dev",
-        "silverstripe/auditor": "2.2.x-dev",
-        "silverstripe/environmentcheck": "2.1.x-dev",
-        "silverstripe/hybridsessions": "2.2.x-dev",
-        "silverstripe/mimevalidator": "2.0.x-dev"
+        "silverstripe/recipe-core": "^4",
+        "cwp/cwp-core": "dev-master",
+        "silverstripe/auditor": "dev-master",
+        "silverstripe/environmentcheck": "dev-master",
+        "silverstripe/hybridsessions": "dev-master",
+        "silverstripe/mimevalidator": "dev-master"
     },
     "require-dev": {
         "phpunit/phpunit": "^5.7"
     },
     "extra": {
-        "branch-alias": {
-            "dev-master": "2.x-dev"
-        },
```

Comment:
Why?

Reply:
dev-master becomes next major, which should be `3.x-dev`.
On the other hand, what's the value of this alias? why do we need it?

Reply:
Because `3.x-dev` is substantially more preferable to `dev-master`, which implies no major release lines at all

Reply:
And this is correct, since we have no plans to do the next major release at the moment.

Reply:
Right, so you should make the `2` branch the default branch and delete the master branch altogether

Reply:
oh, removing master is an interesting idea, we didn't think about it yet